python/pyspark/mllib/tree.py

```diff
@@ -213,21 +227,32 @@ def trainRegressor(cls, data, categoricalFeaturesInfo,
         """
         Train a DecisionTreeModel for regression.
```

Comment:
I slightly prefer making this `decision tree model` or just `decision tree` to be consistent with the forest and GBT doc below.